docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -434,35 +434,35 @@ where t is the period for which transition the is estimated (a year) eg. 365 day
     - Distribution
     - Note
     - Source
-  * - :math:`duration_\text{treated SAM}`
-    - 0-6 months old
+  * - :math:`\text{time to recovery}_\text{effectively treated SAM}`
+    - 0-6 months olds
     - mean: 13.3, sd: 6.9
     - normal
-    -
+    - NOTE: this study reports mean duration of stay in the inpatient therapeutic feeding center rather than time to recovery. It is currently being implemented as time to recovery. Of note, 85% of participants were successfully discharged in this study.
     - [Vygen_2013]_; Niger
-  * - :math:`duration_\text{treated SAM}`
+  * - :math:`\text{time to recovery}_\text{effectively treated SAM}`
     - 6-59 months old
     - 48.3
     - point value
-    -
+    - Studies censored participants who did not recovery in estimation of median time to recovery. 48.3 value is as a weighted average between seven studies reported by [Zw_2020tx]_. Random effects meta analysis would improve this estimate (see todo note below).
     - [Zw_2020tx]_; Ethiopia
-  * - :math:`duration_\text{untreated MAM}`
+  * - :math:`\text{time to recovery}_\text{untreated MAM}`
     - 0-59 months old
     - 63
     - point value
     - IQR reported, but no uncertainty about median value. Used MUAC definition of malnutrition. Value specific to >6 month population, but assumed to generalize to <6 month population.
     - [James_2016]_; Ethiopia
-  * - :math:`duration_\text{treated MAM}`
+  * - :math:`\text{time to recovery}_\text{effectively treated MAM}`
     - 0-6 months old
     - 20.8
     - point value
-    -
+    - NOTE: this study reports mean duration of therapy without specifying if this duration is conditional on recovery. It is currently being implemented as time to recovery. Of note, 81% of participants were successfully discharged in this study.
```

Comment:
When you say it is currently being implemented as time to recovery, does that mean we are treating the duration as being conditional on recovery? What happened to the 19% of patients who were not successfully discharged?

Reply:
Yes, correct. The other 19% either died, failed to respond to treatment, or were lost to follow-up/status unknown.

Reply:
Ok thanks. What is the implication of our interpretation of the duration as time to recovery, given that 81% of patients were successfully discharged?

Reply:
Perhaps to be more explicit about our assumptions, you could say something like:
> NOTE: this study reports mean duration of therapy without specifying if this duration is conditional on recovery. We are currently interpreting the duration as being conditional on recovery and therefore using it as the time to recovery in our model.

And then say something about the 81% successfully discharged, and perhaps mention in the assumptions and limitations what the implications would be if our assumption about it being time-to-recovery were wrong.

Reply:
I think all the same comments apply to `\text{time to recovery}_\text{effectively treated SAM}` above, as it looks like you are making the same assumption there when you say:
> NOTE: this study reports mean duration of stay in the inpatient therapeutic feeding center rather than time to recovery. It is currently being implemented as time to recovery. Of note, 85% of participants were successfully discharged in this study.

Reply:
Yeah, the implications of the 81% successfully discharged is that the majority of patients were successfully discharged (recovered) and the estimate may be skewed by the other 19% who either died or were lost to follow up before recovery, which may decrease the estimate of length of stay. The lower the proportion of patients who were successfully discharged, the greater the potential for bias. 

These studies are in the 0-6 month olds which we are not modeling right now and I plan to address all of these studies in more detail when revisiting the modeling strategy for this age group, but I will add these suggestions now too. Thanks